src/css/base.scss

```diff
@@ -127,6 +132,155 @@ img {
   height: 40%;
 }
 
+//Login SCSS
```

Comment:
I love that you have organization code that tells you what each SCSS is for, that's awesome.